### DISCUSSION TITLE
Can you Group by a field

### DISCUSSION THREAD

**Discussion by Thehunter74** [fbaligand/kibana-enhanced-table#321] [Q&A] [answered]
Hello, Just want to know if it's possible to Group by a field. I am wanting to group by a from field and then display the top hit of subject and verdict along with the count of all messages that from has sent

**Comment by fbaligand** [comment, marked as answer]
Hi,

You should be able to do that:
- First, add a "Bucket", of type "Terms", and choose the wished grouping field
- then, add a "Metric", of type "Top Hit", choose the order to define the top hit, and choose the field that you want to display from the Top Hit
- finally, add a "Metric", of type "Count"

**Reply by Thehunter74** [reply]
Thanks so much!
